web/package.json

```diff
@@ -41,101 +41,100 @@
         "@graphql-codegen/typescript": "4.0.1",
         "@graphql-codegen/typescript-operations": "4.0.1",
         "@graphql-codegen/typescript-react-apollo": "3.3.7",
-        "@playwright/test": "1.33.0",
-        "@rollup/plugin-yaml": "4.1.0",
-        "@storybook/addon-essentials": "7.3.2",
-        "@storybook/addon-styling": "1.3.6",
-        "@storybook/addons": "7.3.2",
-        "@storybook/manager-api": "7.3.2",
-        "@storybook/react": "7.3.2",
-        "@storybook/react-vite": "7.3.2",
-        "@storybook/theming": "7.3.2",
-        "@testing-library/jest-dom": "5.16.5",
+        "@playwright/test": "1.39.0",
+        "@rollup/plugin-yaml": "4.1.2",
+        "@storybook/addon-essentials": "7.5.2",
+        "@storybook/addon-styling": "1.3.7",
+        "@storybook/addons": "7.5.2",
+        "@storybook/manager-api": "7.5.2",
+        "@storybook/react": "7.5.2",
+        "@storybook/react-vite": "7.5.2",
+        "@storybook/theming": "7.5.2",
+        "@testing-library/jest-dom": "6.1.4",
         "@testing-library/react": "14.0.0",
-        "@testing-library/user-event": "14.4.3",
+        "@testing-library/user-event": "14.5.1",
         "@types/apollo-upload-client": "17.0.2",
-        "@types/carbon__colors": "10.31.0",
-        "@types/gapi.auth2": "0.0.57",
-        "@types/gapi.client": "1.0.5",
+        "@types/carbon__colors": "10.31.2",
+        "@types/gapi.auth2": "0.0.59",
+        "@types/gapi.client": "1.0.7",
         "@types/gapi.client.sheets": "4.0.20201031",
-        "@types/google.picker": "0.0.39",
-        "@types/js-md5": "0.7.0",
-        "@types/lodash-es": "4.17.7",
-        "@types/mapbox__vector-tile": "1.3.0",
-        "@types/node": "20.1.4",
+        "@types/google.picker": "0.0.41",
+        "@types/js-md5": "0.7.1",
+        "@types/lodash-es": "4.17.10",
+        "@types/mapbox__vector-tile": "1.3.3",
+        "@types/node": "20.8.10",
         "@types/react": "18.2.6",
         "@types/react-dom": "18.2.4",
         "@types/react-ga": "2.3.0",
         "@types/react-leaflet": "2.8.2",
-        "@types/styled-components": "5.1.26",
-        "@types/tinycolor2": "1.4.3",
-        "@types/uuid": "9.0.1",
-        "@vitejs/plugin-react": "4.0.0",
-        "@vitest/coverage-c8": "0.31.0",
+        "@types/styled-components": "5.1.29",
+        "@types/tinycolor2": "1.4.5",
+        "@types/uuid": "9.0.6",
+        "@vitejs/plugin-react": "4.1.0",
+        "@vitest/coverage-v8": "^0.34.6",
```

Comment:
Can you remove ^ and `vite` again 🙏🏻

Reply:
Fixed! https://github.com/reearth/reearth/pull/783/commits/eb32b0cda2f1297119596e018f11a70548688863